Yoast/yoast-test-helper
# Added grunt without deploy option

## **This PR contains:**

* Grunt commands to create an Artifact folder (+zip)
* Grunt commands to version bump the plugin
## **Test instructions:**
Ensure you have all packages installed by running` yarn`.

Run the `grunt artifact:zip command`, this should generate a new artifact zip, and clean up the /artifact folder.
Run the `grunt artifact command`, this should generate a new artifact zip, but not clean up the /artifact folder.
Run the `grunt set-version --new-version=1.0`, followed up with the `grunt update version command` This should update the versions in all plugin files.

Fixes: #25
